models/tiktok_ads__campaign_country_report.sql

```diff
@@ -0,0 +1,65 @@
+{{ config(enabled=var('ad_reporting__tiktok_ads_enabled', true) and var('tiktok_ads__using_campaign_country_report', true)) }}
+
+with country_report as (
+    
+    select *
+    from {{ var('campaign_country_report') }}
+), 
+
+campaigns as (
+
+    select *
+    from {{ var('campaign_history') }}
+    where is_most_recent_record
+), 
+
+advertiser as (
+
+    select *
+    from {{ var('advertiser') }}
+), 
+
+aggregated as (
+
+    select
+        country_report.source_relation,
+        cast(country_report.stat_time_day as date) as date_day,
```

Comment:
Is there a reason we cast to `date_day` in the transformation model rather than the staging model?

Reply:
No reason, but we should cast at staging model. Made the update in the source package and removing this cast here.